charts/datadog/templates/gke_autopilot_allowlist_synchronizer.yaml

```diff
@@ -1,10 +1,11 @@
-{{- if and .Values.providers.gke.autopilot (eq (include "gke-autopilot-workloadallowlists-enabled" .) "true")}}
+{{- if and .Values.providers.gke.autopilot (eq (include "gke-autopilot-workloadallowlists-enabled" .) "true") (not .Values.datadog.envDict.HELM_FORCE_RENDER) }}
```

Comment:
Don't render this for the CI because the vanilla k8s clusters will not have the `AllowlistSynchronizer` CRD installed.